inc/GameState.h

```diff
@@ -0,0 +1,45 @@
+//
+// Created by danilo.bezerra on 15/12/2023.
+//
+
+#ifndef UNO_GAME_GAMESTATE_H
+#define UNO_GAME_GAMESTATE_H
+
+#include <memory>
+#include <vector>
+#include <map>
+
+#include "Card.h"
+
+class GameState {
+    std::vector<std::shared_ptr<Card>> drawPile;
```

Comment:
What's the difference between GameState drawPile and GameMatch drawPile?

Reply:
I hadn't yet decided where it was going to be, so it's still in the two for testing (I haven't actually decided yet). It's not even represented in the diagram yet.